site/layouts/shortcodes/deprecated-in.html

```diff
@@ -0,0 +1,6 @@
+{{- /* Outputs badge to identify the version something was Deprecated */ -}}
+
+{{- $version := .Get 0 -}}
+
+<!-- Boosted mod: use tag component -->
+<p><span class="tag tag-sm">Deprecated in v{{ $version }}</span></p>
```

Comment:
They look almost the same as the new ones, can we tweak them a bit ? 😓

Reply:
I let you ask to the designers if we can get around the rule and maybe have a danger border for this one.